numpy/f2py/rules.py

```diff
@@ -55,6 +55,9 @@
 from . import __version__
 f2py_version = __version__.version
 
+from .. import version as _numpy_version
+numpy_version = str(_numpy_version.version)
```

Comment:
Why the conversion to `str`? If `version.version` is not a string this should probably just fail

Reply:
Ok, let us just have the string concatenation fail with TypeError if version.version is e.g. changed to a tupe in the future.